<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportPage</class>
 <widget class="QWidget" name="ImportPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="wizardGroup">
     <property name="title">
      <string>Morrowind Installation Wizard</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1">
      <item row="1" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="wizardButton">
        <property name="text">
         <string>Run &amp;Installation Wizard</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="importerGroup">
     <property name="title">
      <string>Morrowind Settings Importer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="1,0">
        <item row="0" column="0">
         <widget class="QLabel" name="importerLabel">
          <property name="text">
           <string>File to Import Settings From:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="settingsComboBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="addonsCheckBox">
        <property name="text">
         <string>Import Add-on and Plugin Selection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fontsCheckBox">
        <property name="toolTip">
         <string>Fonts shipped with the original engine are blurry with UI scaling and support only a small amount of characters,
so OpenMW provides another set of fonts to avoid these issues. These fonts use TrueType technology and are quite similar
to default Morrowind fonts. Check this box if you still prefer original fonts over OpenMW ones or if you use custom bitmap fonts.</string>
        </property>
        <property name="text">
         <string>Import Bitmap Fonts</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1">
        <item row="2" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="importerButton">
          <property name="text">
           <string>Run &amp;Settings Importer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximum">
         <number>4</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
